tests/queries/0_stateless/02833_local_with_dialect.reference

```diff
@@ -0,0 +1,2 @@
+0
+[?2004h[?2004lBye.
```

Comment:
what exactly is that in front of `Bye`, i.e. can it be flaky when used in test?

Reply:
I piped the output of `clickhouse-client` into this file and I haven't put too effort into understanding it. However this was always the case, so I guess it is not flaky.